src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -225,6 +237,84 @@ export default {
 			// 		},
 			// 	});
 			// }
+		},
+		fetchRemainingLoanChannels() {
+			const ssrLoanIds = [];
+			// pick loan ids from each
+			_each(this.categories, category => {
+				_each(category.loans.values, loan => {
+					ssrLoanIds.push(loan.id);
+				});
+			});
+			// Client Fetch the remaining category rows
+			return this.apollo.query({
+				query: loanChannelQuery,
+				variables: {
+					ids: _drop(this.realCategoryIds, ssrRowLimiter),
+					excludeIds: ssrLoanIds,
+					// @todo variables for fetching data for custom channels
+				},
+			}).then(({ data }) => {
+				// add our remaining loan channels
+				this.clientCategories = _get(data, 'lend.loanChannelsById') || [];
+				Promise.resolve();
+			});
+		},
+		fetchRecentlyViewed() {
+			// Setup Recently Viewed Loans data for inclusion in page load loan row analytics
+			// Read assignment for Recently Viewed Loans EXP
+			const recentlyViewedEXP = this.apollo.readQuery({
+				query: experimentQuery,
+				variables: { id: 'recently_viewed_loans' }
+			});
+			this.showRecentlyViewed = _get(recentlyViewedEXP, 'experiment.version') === 'variant-a';
+			// if Recently Viewed Exp is active look for loans in local storage
+			if (this.showRecentlyViewed) {
+				// fetch recently viewed from localStorage (currently set in wwwApp on Borrower Profile)
+				const recentlyViewed = WebStorage('recentlyViewedLoans');
+				// decode, parse then set recently viewed loan data
+				try {
+					this.recentLoanIds = JSON.parse(atob(recentlyViewed));
+				} catch (e) {
+					// no-op
+				}
+			}
+		},
+		activateWatchers() {
```

Comment:
Moving these out to mounted and after we've fetched all loans eliminated the need to do multiple versions for the prefetch/ssr and client sets of loans.

Reply:
Had to preserve these watchers to ensure the loans updated after basket update. May be able to whittle them down more...